src/mmw/js/src/modeling/templates/projectMenu.html

```diff
@@ -1,46 +1,53 @@
-<div class="project-title">
-    <button class="btn dropdown-toggle project-dropdown" type="button" data-toggle="dropdown" aria-expanded="true">
-        {{ name }}
-        {% if editable or is_new or itsi %}
-            <i class="fa fa-caret-down"></i>
-        {% endif %}
-    </button>
-    {% if editable or is_new or itsi %}
-        <ul class="dropdown-menu menu-right" role="menu">
-            {% if not is_new and editable %}
-                <li role="presentation"><a role="menuitem" tabindex="-1" data-toggle="modal" id="share-project">Share</a></li>
-                <li role="presentation"><a role="menuitem" tabindex="-1" data-toggle="modal" id="project-privacy">{% if is_private %}Make Public{% else %}Make Private{% endif %}</a></li>
-                <li role="presentation"><a role="menuitem" tabindex="-1" data-toggle="modal" id="delete-project">Delete</a></li>
-            {% endif %}
-            {% if editable %}
-                <li role="presentation"><a role="menuitem" tabindex="-1" data-toggle="modal" id="rename-project">Rename</a></li>
-            {% endif %}
-            {% if is_new %}
-                <li role="presentation"><a role="menuitem" tabindex="-1" id="save-project">Save</a></li>
+<div class="project-left">
+    <div class="project-title">
+        <button class="btn dropdown-toggle project-dropdown" type="button" data-toggle="dropdown" aria-expanded="true">
+            {{ name }}
+            {% if editable or is_new or itsi %}
+                <i class="fa fa-caret-down"></i>
             {% endif %}
-            {% if itsi %}
-                <li role="presentation"><a role="menuitem" tabindex="-1" id="itsi-clone">Embed in ITSI</a></li>
-            {% endif %}
-            {% if user and not itsi_embed %}
-                <li role="separator" class="divider"></li>
-                <li role="presentation"><a role="menuitem" tabindex="-1" data-url="/projects/">My Projects</a></li>
+        </button>
+        {% if editable or is_new or itsi %}
+            <ul class="dropdown-menu menu-right" role="menu">
+                {% if not is_new and editable %}
+                    <li role="presentation"><a role="menuitem" tabindex="-1" data-toggle="modal" id="share-project">Share</a></li>
```

Comment:
Consider breaking these element tags across several lines, one tag per line, to make this more legible.